core/embed/upymod/modtrezorutils/modtrezorutils.c

```diff
@@ -356,18 +360,17 @@ STATIC mp_obj_t mod_trezorutils_check_heap_fragmentation(void) {
   size_t main_map_alloc = MP_STATE_VM(dict_main).map.alloc;
 #endif
   if (main_map_alloc > MICROPY_MAIN_DICT_SIZE) {
-    mp_raise_msg(&mp_type_AssertionError, "main globals dict is reallocated");
+    mp_raise_msg(&mp_type_AssertionError,
+                 MP_ERROR_TEXT("main globals dict is reallocated"));
   }
 
   size_t n_pool, n_qstr, n_str_data_bytes, n_total_bytes;
   qstr_pool_info(&n_pool, &n_qstr, &n_str_data_bytes, &n_total_bytes);
   if (n_pool) {
     qstr_dump_data();
     mp_raise_msg_varg(&mp_type_AssertionError,
-                      "Runtime QSTR allocation: " UINT_FMT " pools, " UINT_FMT
-                      " strings, " UINT_FMT " data bytes, " UINT_FMT
-                      " total bytes",
-                      n_pool, n_qstr, n_str_data_bytes, n_total_bytes);
+                      MP_ERROR_TEXT("Runtime QSTR allocation detected"), n_pool,
+                      n_qstr, n_str_data_bytes, n_total_bytes);
```

Comment:
Does this still work without the `UINT_FMT`s?

Reply:
It doesn't :(
Fixed in https://github.com/trezor/trezor-firmware/pull/4961/commits/d26fb6a9726cb43f064580b4d1723767ebf1d258.